core/finance/financial_metrics.py

```diff
@@ -44,8 +46,9 @@ def get_metrics(self, pronac, metrics=None):
                 print('Getting Metrics for [{}]'.format(metric))
                 metric_obj = self.metrics[metric]
                 results[metric] = metric_obj.get_metrics(pronac)
-            except: #TODO: create exception types
-                pass
+            except DataNotFoundForPronac as ex:
```

Comment:
Já que a variável `ex` não está sendo utilizada, sugiro trocar a linha por:

```suggestion
            except DataNotFoundForPronac:
```